bca (1).md

```diff
@@ -0,0 +1,149 @@
+### Create BCA.txt. Write 5 lines directly and 5 lines must be taken from user. Print the smallest and largest word in this file.
+
+
+```python
+f=open("C:/23bca280/python/BCA.txt","w")
```

Comment:
So we can open this file in another way.

Reply:
yes, we can!
we can open this by with statement and open() function

Reply:
ok ,thanks for understand me this topic